### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by BenjaminMatembe**
**In this feature, I have worked on the following  👍** 

✔️ Initialised react app
✔️ Setup Linters
✔️ Added a descriptive Readme.md

### CODE REVIEW

**Review by Davidosky007 — COMMENTED** [review]
Hi @BenjaminMatembe 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have a descriptive readme, Kindly Add descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1 +1,258 @@
...
+
+### `npm run build` fails to minify
+
+This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+
+
+<a name="readme-top"></a>
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
```
Comment (Olamarx) [inline README.md]: - [OPTIONAL] I will encourage you to remove this as pect since they are default data from CRA.

**Review by Olamarx — APPROVED** [review]
Hi @BenjaminMatembe, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Linters passed ✔️
- Commit messages meaningful ✔️
- Readme file professional ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by BenjaminMatembe** [comment]
Hi, @Olamarx thank you for the advise and approval of my project.
I will definitely consider the optional suggestions.

**Pull request merged**